docs/concepts/ccd.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Customer Cloud Deployments
+hide_table_of_contents: false
+---
+[console]: https://console.strmprivacy.io
+[cli]: https://docs.strmprivacy.io/docs/latest/cli-index/
+[api]: https://github.com/strmprivacy/api-definitions/tree/master/protos/strmprivacy
+
+In a Customer Cloud Deployment (CCD), _all event processing takes place in a Kubernetes cluster specified by the
+customer_. **Your sensitive events never leave your own cluster.**
+
+The entities doing the processing get configured by Strmprivacy controls _outside that cluster_, in the Strmprivacy
+cloud. Inside the _customer data plane_ several _agents_ are running that query `api.strmprivacy.io` every few seconds.
+
```

Comment:
Maybe add an extra sentence that this is part of our control plane/management layer and only contains configuration data. Maybe even add a graphic as well to make it absolutely clear that there is a separation of data plane and control plane.